lib/logdna.rb

```diff
@@ -15,11 +15,11 @@ class Ruby < ::Logger
     # uncomment line below and line 3 to enforce singleton
     # include Singleton
     Logger::TRACE = 5
-    attr_accessor :level, :app, :env, :meta
+    attr_accessor :log_level, :app, :env, :meta
```

Comment:
Let's just remove `log_level` from public access altogether; i.e. remove it from this list.